HotFix.Test/utilities/scheduling/get_active.cs

```diff
@@ -0,0 +1,124 @@
+﻿using System;
+using System.Collections.Generic;
+using FluentAssertions;
+using HotFix.Core;
+using HotFix.Utilities;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using static System.DayOfWeek;
+
+namespace HotFix.Test.utilities.scheduling
+{
+    [TestClass]
+    public class get_active
```

Comment:
Since we decided not to properly name these tests, it would be a bit clearer to go with something like `getting_the_active_scheduled_session`, and then calling the tests `scenario_a`, `scenario_b` etc.